src/config/options.schema.json

```diff
@@ -534,7 +534,7 @@
                 "type": "string",
                 "enum": [
                   "congruence",
-                  "singleThreaded",
+                  "reduceAnalyses",
```

Comment:
After some back-and-forth, this still just does the single threaded thing and the race analyses are instead part of `concurrencySafetySpecification`, right?